maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -170,6 +173,22 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun screenshot(
+        request: MaestroAndroid.ScreenshotRequest,
+        responseObserver: StreamObserver<MaestroAndroid.ScreenshotResponse>
+    ) {
+        val outputStream = ByteString.newOutput()
+        val bitmap = uiAutomation.takeScreenshot()
+        if (bitmap.compress(Bitmap.CompressFormat.PNG, 100, outputStream)) {
+            responseObserver.onNext(screenshotResponse {
+                bytes = outputStream.toByteString()
+            })
+            responseObserver.onCompleted()
+        } else {
+            responseObserver.onError(Throwable("Failed to compress bitmap"))
```

Comment:
Lets log here as well since we can use it debug from logs of maestro server apk

Reply:
Updated